submissions/irynatsarenok/pop_up/pop_up.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="pop_up.css" />
+    <title>POPUP</title>
+  </head>
+  <body>
+    <header>
+      <a href="#" class="header_link link focus_item">
+        <h5>Mail</h5>
+      </a>
+
+      <a href="#" class="header_link link focus_item">
+        <h5>Images</h5>
+      </a>
+
+      <div>
+        <input type="checkbox" id="hooli" class="checkbox" />
+        <label for="hooli" class="popup_label">
+          <img src="pop_up_icons/popup-button.png" width="25" height="25" />
+        </label>
+        <div class="popup invisible-scrollbar">
+          <div class="popup_div">
+            <a href="#" class="popup_link link">
+              <img
+                src="pop_up_icons/calendar.png"
+                alt="Calendar"
+                width="55"
+                height="40"
+                class="popup_img"
+              />
+              Calendar
+            </a>
+            <a href="#" class="popup_link link">
+              <img
+                src="pop_up_icons/mail.png"
+                alt="Mail"
+                width="55"
+                height="40"
+                class="popup_img"
+              />
+              Mail
+            </a>
+            <a href="#" class="popup_link link">
+              <img
+                src="pop_up_icons/news.png"
+                alt="News"
+                width="55"
+                height="40"
+                class="popup_img"
+              />
+              News
+            </a>
+          </div>
+
+          <div class="popup_div">
```

Comment:
Use [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [gid](https://css-tricks.com/snippets/css/complete-guide-grid/) to define element grid.